### PR TITLE
Enrolled myself

### PR DESCRIPTION
**Pull request by mumairofficial**

### CODE REVIEW

**Comment by sam-cross** [comment]
Country flag is wrong, have a look at the check logs

File: src/data.js [hunk 1]
```diff
@@ -2064,6 +2064,27 @@ const pages = [
...
     ],
   },
+  {
+    name: 'Muhammad Umair',
+    description:
+      'Fullstack front-end developer and designer, passionate in everything #JavaScript',
+    url: 'https://www.linkedin.com/in/mumairofficial',
+    twitter: '@imumare',
+    emoji: '💻',
+    country: 'PK',
```
Comment (mumairofficial) [inline src/data.js]: I see PK is there in supported coutries list;

image

Comment (mumairofficial) [reply src/data.js]: it's also available in https://github.com/wesbos/awesome-uses/blob/93b8ad6856184c4b80406b1c3c09b5c4bc542167/scripts/flags.js#L163

Comment (palashmon) [reply src/data.js]: It's actually unicode emoji code `🇵🇰` not `PK`. try to copy-paste from `flags.js` once.

**Comment by mumairofficial** [comment]
@sam-cross It was my bad, @palashmon thank you for quick hint. I have fixed the problem

File: src/data.js [hunk 2]
```diff
@@ -2064,6 +2064,27 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Muhammad Umair',
+    description:
+      'Fullstack front-end developer and designer, passionate in everything #JavaScript',
+    url: 'https://www.linkedin.com/in/mumairofficial',
```
Comment (palashmon) [inline src/data.js]: I think there is one more issue here. This [`uses.tech`](https://uses.tech/) site actually shows a list of `/uses` pages detailing developer setups, gear, software and configs like:

```
https://wesbos.com/uses
https://farrant.me/uses
https://codewithhugo.com/uses/
```

but the URL you have shared here is just your LinkedIn profile. Please add your `/uses` page url here, if you have any. Thanks!

Comment (HugoDF) [reply src/data.js]: I second this, setting this to "request changes"

Comment (mumairofficial) [reply src/data.js]: @HugoDF I have created gist for now. Thank you for the idea; mean while I am developing my personal portfolio site!

**Comment by HugoDF** [comment]
@mumairofficial we'll need to integrate master changes

**Comment by mumairofficial** [comment]
Something went wrong, I going to close thisPR request and create new one.

**Pull request closed without merging**